test/libsolidity/semanticTests/calldata/copy_from_calldata_removes_bytes_data.sol

```diff
@@ -0,0 +1,10 @@
+contract c {
+    function set() public returns (bool) { data = msg.data; return true; }
+    fallback() external { data = msg.data; }
+    bytes data;
+}
+// ====
+// compileViaYul: also
+// ----
+// set(): 1, 2, 3, 4, 5 -> true
+// gas legacy: 177656
```

Comment:
The original test calls the `fallback` here.

Reply:
Update : Added relevant code to the test case to make it call the fallback()